packages/core/src/models/LibrariesState.ts

```diff
@@ -1,9 +1,10 @@
+import { SharedSliceContent } from "@prismicio/types-internal/lib/content/fields/slices/SharedSliceContent";
 import { SharedSlice } from "@prismicio/types-internal/lib/customtypes/widgets/slices";
 import * as t from "io-ts";
 // import { VariationMock } from "./Slice";
```

Comment:
nice, now the comments can be removed, (they where there to remind me about removing valiatio mock)